transbank/oneclick/transaction_mall.py

```diff
@@ -0,0 +1,79 @@
+import requests
```

Comment:
Name of the file should be mall_transaction.py

Reply:
done